### DISCUSSION TITLE
Configuring Cookie Attributes?

### DISCUSSION THREAD

**Discussion by mdodell** [janko/rodauth-rails#134] [Q&A] [answered]
Is it possible to configure the [cookies attributes](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie) when using Rodauth as a JSON API outlined in [this tutorial](https://github.com/janko/rodauth-rails/wiki/JSON-API#json-cookie-based)?

**Comment by janko** [comment, marked as answer]
Rodauth with `json` feature enabled is using Rails session, just like a non-API application, which Rails stores in the cookie by default (Rodauth doesn't use a separate cookie for this). Cookie attributes for the Rails session can be configured [via Rails config](https://api.rubyonrails.org/classes/ActionDispatch/Session/CookieStore.html).
